selftests/unit/test_utils_params.py

```diff
@@ -86,6 +86,96 @@ def testGetItemMissing(self):
     def testGetItem(self):
         self.assertEqual(self.params['image_size'], "10G")
 
+    def testCfgEnabled(self):
```

Comment:
I like this method

Reply:
It was renamed to `get_boolean` after @luckyh's suggestion. Also updated unit tests.